clients/js/test/printV1.test.ts

```diff
@@ -88,14 +96,19 @@ test('it can print a new edition from a ProgrammableNonFungible', async (t) => {
   // When we print a new edition of the asset.
   const editionMint = generateSigner(umi);
   const editionOwner = generateSigner(umi);
-  await printV1(umi, {
-    masterTokenAccountOwner: originalOwner,
-    masterEditionMint: originalMint.publicKey,
-    editionMint,
-    editionTokenAccountOwner: editionOwner.publicKey,
-    editionNumber: 1,
-    tokenStandard: TokenStandard.ProgrammableNonFungible,
-  }).sendAndConfirm(umi);
+  await transactionBuilder()
+    .add(setComputeUnitLimit(umi, { units: 400_000 }))
```

Comment:
It is a case of failing sometimes due to derivation asserts compute variance, which is normal since we are using random public keys. Settings a higher compute units makes sure that it will always pass.